route53_crossfeed_app.tf

```diff
@@ -300,13 +310,23 @@ resource "aws_route53_record" "crossfeed_staging_api_CNAME1" {
 resource "aws_route53_record" "crossfeed_staging_api_CNAME2" {
   provider = aws.route53resourcechange
 
-  name    = "_95c4bc30d3e15e6b58dc96250e3661a4.api.staging.crossfeed.${aws_route53_zone.cyber_dhs_gov.name}"
+  name    = "staging-cd.api.crossfeed.${aws_route53_zone.cyber_dhs_gov.name}"
   records = ["_73aacddb56ac20ef77492c962bf006d8.jfrzftwwjs.acm-validations.aws"]
   ttl     = 300
   type    = "CNAME"
   zone_id = aws_route53_zone.cyber_dhs_gov.zone_id
 }
 
+resource "aws_route53_record" "crossfeed_staging_api_CNAME3" {
+  provider = aws.route53resourcechange
+
+  name    = "staging-cd.api.crossfeed.${aws_route53_zone.cyber_dhs_gov.name}"
```

Comment:
Should this be staging.api?

Reply:
@epicfaace Looks like this was taken care of in https://github.com/cisagov/cool-dns-cyber.dhs.gov/pull/76/commits/6876b587812f31dd4991babc3475da91df0a98c7.  If so, please resolve this conversation.